provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/ProvisioningServiceClient.java

```diff
@@ -160,7 +167,7 @@ private ProvisioningServiceClient(String connectionString)
      * {
      *     // Create a Device Provisioning Service Client.
      *     DeviceProvisioningServiceClient deviceProvisioningServiceClient =
-     *         DeviceProvisioningServiceClient.createFromConnectionString(PROVISIONING_CONNECTION_STRING);
```

Comment:
Since this is a major version bump and can have breaking changes, are we planning on deleting `DeviceProvisioningServiceClient.createFromConnectionString` anyway?
If not, do we plan on marking it as deprecated here instead. 

I think having multiple ways to initialize a client with the same parameters can be confusing to customers, and we should provide them with a clear recommended path. What do you think?

Reply:
I'm only hesitant to delete it now because it was never deprecated. As per the point @vinagesh makes below, I'd be happy to mark this static constructor as deprecated to help cut down on the confusion here.